Mobile/lib/pages/Register.dart

```diff
@@ -30,79 +32,89 @@ class _RegisterState extends State<Register> {
-          key: _formKey,
-          child: Column(
-            mainAxisAlignment: MainAxisAlignment.center,
-            children: [
-              TextFormField(
-                decoration: InputDecoration(labelText: 'Name'),
-                validator: (value) {
-                  if (value == null || value.isEmpty) {
-                    return 'Name is required and cannot be empty';
-                  }
-                  return null;
-                },
-                onSaved: (value) {
-                  _name = value!;
-                },
-              ),
-              SizedBox(height: 10),
-              TextFormField(
-                decoration: InputDecoration(labelText: 'Email'),
-                keyboardType: TextInputType.emailAddress,
-                validator: (value) {
-                  if (value == null || value.isEmpty) {
-                    return 'Email is required and cannot be empty';
-                  }
-                  if (!RegExp(r'^[^@]+@[^@]+\.[^@]+').hasMatch(value)) {
-                    return 'The entered email is not a valid email';
-                  }
-                  return null;
-                },
-                onSaved: (value) {
-                  _email = value!;
-                },
-              ),
-              SizedBox(height: 10),
-              TextFormField(
-                decoration: InputDecoration(labelText: 'Password'),
-                obscureText: true,
-                validator: (value) {
-                  if (value == null || value.isEmpty) {
-                    return 'Password is required and cannot be empty';
-                  }
-                  return null;
-                },
-                onChanged: (value) {
-                  _password = value;
-                },
-              ),
-              SizedBox(height: 10),
-              TextFormField(
-                decoration: InputDecoration(labelText: 'Password Confirmation'),
-                obscureText: true,
-                validator: (value) {
-                  if (_password != value) {
-                    return 'Password does not match';
-                  }
-                  return null;
-                },
-                onSaved: (value) {
-                  _password_confirmation = value!;
-                },
-              ),
-              SizedBox(height: 30),
-              ElevatedButton(
-                onPressed: tryRegister,
-                child: Text('Create account'),
-              ),
-            ],
+      body: Stack(children: [
+        MeshGradientBackground(),
+        Padding(
+          padding: const EdgeInsets.all(20),
+          child: Form(
+            key: _formKey,
+            child: Column(
+              mainAxisAlignment: MainAxisAlignment.center,
+              children: [
+                Image.file(
+                  File('assets\\Logo.png'),
+                  height: 150,
+                  fit: BoxFit.cover,
+                ),

```

Comment:
Image.file doesn't seem to work correctly on an android phone.
I suggest using Image.asset instead as to my testing it works better.
I would also suggest using `/` instead of `\\` as double backslash is only on windows, most other systems uses a single forward slash.
```suggestion
                Image.asset(
                  'assets/Logo.png',
                  height: 150,
                  fit: BoxFit.cover,
                ),

```